Libraries/SDHC/ff13/Source/diskio.h

```diff
@@ -40,7 +42,7 @@ DRESULT disk_read (BYTE pdrv, BYTE* buff, DWORD sector, UINT count);
 DRESULT disk_write (BYTE pdrv, const BYTE* buff, DWORD sector, UINT count);
 DRESULT disk_ioctl (BYTE pdrv, BYTE cmd, void* buff);
 DWORD get_fattime(void);
-
+void disk_flash(uint32_t isFlash);
```

Comment:
`diskio.h` modification - adding this function to the API should only be done as an absolute last resort.  Additionally, `uint32_t` is used which goes against the FatFS convention of using its built-in data types.

Reply:
I am not sure how I can come around to not changing the diskio.h for this code snippet. I will look into it.

Reply:
See https://github.com/Analog-Devices-MSDK/msdk/pull/571/files#r1184416416